src/ProjectTemplates/Web.ProjectTemplates/content/RazorClassLibrary-CSharp/.template.config/fr-FR/strings.json

```diff
@@ -0,0 +1,7 @@
+{
+  "version": "1.0.0.0",
+  "strings": {
+    "name": "Bibliothèque de classes Razor",
+    "description": "Modèle de projet pour créer une bibliothèque de classes Razor."
```

Comment:
My French isn't great. But, I'm wondering whether the translation of "library" (as in a collection of classes) to "bibliothèque" is more literal than it should be. Probably fine&hellip;

Reply:
All from the loc team and what we already have in VS.. these are not new translations.